### DISCUSSION TITLE
Dynamic routes "places-in-[...stub].tsx"

### DISCUSSION THREAD

**Discussion by NCharby** [vercel/next.js#24671] [Help] [answered]
Hey folks! 

I have a need to support a large number of routes with a place name, such as:
`mysite.com/places-in-atlanta`
`mysite.com/places-in-new-york`
`mysite.com/places-in-san-diego`

I'm attempting to define a route `/pages/places-in-[...stub].tsx`, but this does not catch the requests.
Unfortunately this needs to be on the root to support permalinks from an older project, so `/pages/places/[[...stub]].tsx` is not an option.

Is this possible in Next without the use of a custom server?

**Comment by laugharn** [comment, marked as answer]
Your best bet will be to use a rewrite in next.config.js with a pattern match:

```js
rewrites() {
  return {
    fallback: [
      {
        destination: '/places-in/:place',
        source: '/:path(places-in-[a-zA-Z-]{1,})',
      },
    ],
  },
}
```

From there you can use `/places-in/[place].tsx`, and just trim the 'places-in-' prefix from `params.place`
